lib/ElementHandle.js

```diff
@@ -118,7 +118,7 @@ class ElementHandle extends JSHandle {
   }
 
   /**
-   * @return {!Promise<?{x: number, y: number, width: number, height: number}>}
+   * @return {!Promise<?{border: object, content: object, margin: object, padding: object, width: number, height: number}>}
    */
   async boundingBox() {
```

Comment:
Can we rename this into `boxModel`? The `boundingBox` was named to match the `getBoundingClientRect` semantics; the new behavior no longer matches it.